server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -224,6 +227,15 @@ public Response validate(Message command) {
         return result;
     }
 
+    private void schedule(Command command) {
+        if (scheduler != null) {
+            scheduler.schedule(command);
+        } else {
+            throw new IllegalStateException(
+                    "Scheduled commands are not supported by this command bus (scheduler is not set).");
```

Comment:
Consider having two sentences in the message, instead of having the nested one in braces.